blueflood-http/src/test/resources/sample_enums_payload.json

```diff
@@ -1,6 +1,6 @@
 {
     "tenantId":"333333",
-    "timestamp":1439231324000,
+    "timestamp":1389211230,
```

Comment:
@VinnyQ any reason why you changed the ts and made it seconds instead of millis?